packages/forms/src/model.ts

```diff
@@ -1323,8 +1362,9 @@ export class FormControl extends AbstractControl {
   }
 
   /**
-   * Resets the form control, marking it `pristine` and `untouched`, and setting
-   * the value to null.
+   * Resets the form control, marking it `pristine` and `untouched`, and resetting
+   * the value. The new will be the provided value (if passed), `null`, or the initial value if
```

Comment:
Looks like a typo in the sentence?
```suggestion
   * the value. The new value will be the provided one (if passed), `null`, or the initial value if
```

Reply:
Fixed.